bookwyrm/views/feed.py

```diff
@@ -52,6 +52,8 @@ def get(self, request, tab, filters_applied=False):
 
         suggestions = suggested_users.get_suggestions(request.user)
 
+        readthroughs = models.ReadThrough.objects.filter(user=request.user)
```

Comment:
This works, but as written it means if a new user "starts reading" a book they will suddenly have a `ReadThrough` and "year in the books" will appear. You could do something like below to ensure we are only checking for readthroughs that were "last year" or earlier:

```py
from datetime import date # line 2

cutoff = date(get_annual_summary_year(), 12, 31) # line 55
readthroughs = models.ReadThrough.objects.filter(
    user=request.user, 
    finish_date__lte=cutoff
    )
```